views/peeps.erb

```diff
@@ -0,0 +1,33 @@
+<style>
```

Comment:
Appreciate that you took time to style your app. You can take a screenshot of the page and add it to your readme to give a quick sneak peek.